awx/settings/defaults.py

```diff
@@ -165,6 +165,11 @@
 # reverse proxy.
 REMOTE_HOST_HEADERS = ['REMOTE_ADDR', 'REMOTE_HOST']
 
+URL_REPLACE_FROM_HEADER = {
+    'header': 'Aap-Prefix',  # e.g. api/controller/v2
+    'to_replace': 'api/v2',  # the header string will replace this substring in the URL
```

Comment:
this would be a great thing to test using `override_settings`

Reply:
No wait, we don't care about different setting values. We want to test different headers. Not sure if this is doable in unit tests now.